plyer/platforms/macosx/libs/osx_paths.py

```diff
@@ -1,11 +1,14 @@
 import ctypes
 import os
 
+LibraryPath = "/System/Library/Frameworks/CoreFoundation.framework/"
+"Versions/A/CoreFoundation"
```

Comment:
Does this work? i'm pretty sure the second line won't be bart of the string.

```
>>> a = "aaa"
>>> "bbb"
'bbb'
>>> print(a)
aaa
```
but you could just add a `\` at the end of the first line.
```
>>> a = "aaa"\
... "bbb"
>>> print(a)
aaabbb
```

Reply:
ok